BaseTools/Conf/tools_def.template

```diff
@@ -895,6 +902,8 @@ NOOPT_*_*_OBJCOPY_ADDDEBUGFLAG     = --add-gnu-debuglink="$(DEBUG_DIR)/$(MODULE_
 *_*_*_DTC_PATH                     = DEF(DTC_BIN)
 
 DEFINE GCC_ALL_CC_FLAGS            = -g -Os -fshort-wchar -fno-builtin -fno-strict-aliasing -Wall -Werror -Wno-array-bounds -include AutoGen.h -fno-common -fstack-protector -mstack-protector-guard=global
+DEFINE GCC_IA32_CC_FLAGS           = DEF(GCC_ALL_CC_FLAGS) -m32 -malign-double -freorder-blocks -freorder-blocks-and-partition -O2 -mno-stack-arg-probe # MU_CHANGE
+DEFINE GCC_X64_CC_FLAGS            = DEF(GCC_ALL_CC_FLAGS) -mno-red-zone -Wno-address -mno-stack-arg-probe # MU_CHANGE

```

Comment:
I don't think these two actually get used anywhere, we can drop?

Reply:
These two would be used by any `GCC` `IA32` and `X64` compiles. The `GCC_IA32_` and `GCC_X64_` portions just tell the build system when to apply these flags to any `CC` command (i.e. when the compiler is `GCC` and the arch is `IA32` or `X64`

Reply:
nevermind this is a `DEFINE`. you are right.

Reply:
dropped